app/models/apple/api.rb

```diff
@@ -183,7 +183,7 @@ def unwrap_bridge_response(resp)
 
     def bridge_remote(bridge_resource, bridge_options, batch_size: DEFAULT_BATCH_SIZE)
       url = bridge_url
-      Rails.logger.info("Apple::Api BRIDGE #{bridge_resource} #{url.hostname}:#{url.port}/bridge")
+      Rails.logger.info("Apple::Api BRIDGE #{bridge_resource} #{url.hostname}:#{url.port}/bridge", {param_count: bridge_options.count})
```

Comment:
Digging through the calls to this method a bit ... so `bridge_options` is really the _operations_ we're calling the bridge to perform?  So `bridge_options.count` is a measure of how much work we're calling the bridge with?

Reply:
That's right.  The api requests are batched as a set of options.